### ISSUE TITLE
Add automated roll() function to DataArray and Dataset 

### ISSUE THREAD

**Issue by fmaussion** [fmaussion/salem#72]
Based on http://xarray.pydata.org/en/stable/generated/xarray.DataArray.roll.html, it should be quite easy to implement a function wich automatically converts [-180, 180] datasets to [0,360] and vice versa

**Comment by darothen** [comment]
Yes, it's quite easy. [Here's a function to convert longitudes in (180, 360\]](https://github.com/darothen/darpy/blob/db829049aed4d846c7635ee29277a3b4524c7dfa/darpy/utilities.py#L308), then you would just use `xarray.DataArray.roll.()` along the longitude dimension, by length(longitude) - 1 units.

**Comment by fmaussion** [comment]
Thanks for the tip @darothen !
